include/sway/input/seat.h

```diff
@@ -220,6 +223,9 @@ bool seat_is_input_allowed(struct sway_seat *seat, struct wlr_surface *surface);
 
 void drag_icon_update_position(struct sway_drag_icon *icon);
 
+enum wlr_edges find_resize_edge(struct sway_container *cont,
```

Comment:
I'm not super big on the semantics of having this function exported from here; if anyone has a better home for it I'm happy to move it.